tests/inline/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1317,12 +1371,17 @@ exports[`mixed-32.php 1`] = `
 <div></div>
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 <div></div>
-<?php if (true); ?>
+<?php if (
+    true
+); ?>
     <?php echo 'test'; ?>
 <div></div>
 <div></div>
-<?php if (true); ?>
-<?php echo 'test'; ?>
+<?php  ?>
+<?php
+if (true);
+echo 'test';
+?>
```

Comment:
This is a regression because the `if` node in
```php
<?php if (true) ?>
```
has `loc.end.line == 2` while 
```php
<?php if (true); ?>
```
has `loc.end.line == 1`.
Looks like a parser bug to me.